h2o-bindings/bin/bindings.py

```diff
@@ -368,15 +368,19 @@ def find_field(fields, field_name):
             # Write the generic information about the base class
             schema = m[base]
             schema["generics"] = generics
-            generic_map = {long_type: gen_type for gen_type, long_type in generics}
+            generic_map = defaultdict(list)
+            for name, schema_name in generics:
+                generic_map[schema_name].append(name)     
             generic_index = {geninfo[0]: i for i, geninfo in enumerate(generics)}
             mapped_fields = {}
             for field in schema["fields"]:
                 ftype = field["schema_name"]
                 if ftype in generic_map:
                     gen_type = generic_map[ftype]
-                    field["schema_name"] = gen_type
-                    mapped_fields[field["name"]] = generic_index[gen_type]
+                    if len(gen_type) > 1:
+                        gen_type.pop(0) 
+                    field["schema_name"] = gen_type[0]
+                    mapped_fields[field["name"]] = generic_index[gen_type[0]]
```

Comment:
you changed the logic here, didn't you?
You used to `pop` the first element after reading it from `gen_type` (which made sense).
Now you remove it, and then read the new head element... so you'll never read the original head: looks like a bug.

Reply:
actually I realize that I don't understand why you had to change anything in this file, maybe it was necessary in previous phase of your PR, but I don't see the need now.
If we look at the mapping defined at lines 363-366, it hasn't changed, so the old `generic_map` should still work... it use to produce sth like:
```
dict(ModelParametersSchemaV3="P", ModelOutputSchemaV3="O")
```
and now it produces:
```
dict(ModelParametersSchemaV3=["P"], ModelOutputSchemaV3=["O"])
```
No benefit, and the `len(gen_type)` will never be > 1

Reply:
Yes, it's on the wrong place here.. regarding whole generic map I think after this commit https://github.com/h2oai/h2o-3/pull/4361/commits/f85be8d5a4af114d2f61fb8b3d99df3170c92868 any changes in bindings.py are not needed anymore, so will revert this